main/helpers.py

```diff
@@ -0,0 +1,38 @@
+from .models import Player, Field, Match
+
+
+def get_invalid_matches():
+    matches = Match.objects.all()
+    invalid_match_ids = [match.id for match in matches if is_match_invalid(match)]
+    return invalid_match_ids
+
+
+def is_match_invalid(match):
+    score_list = [match.player2_score, match.player1_score]
+    if score_list.count(11) == 1:
+        if 0 <= match.player1_score <= 11 and 0 <= match.player2_score <= 11:
```

Comment:
you could simply return !(0 <= match.player1_score <= 11 and 0 <= match.player2_score <= 11) if you don't plan to add more stuff to your two branches

Reply:
I agree that this logic could be written in a simpler way - @thomasbean-prezi , care to sit down with @ochronus for 10 minutes to come up with something short and sweet?

Reply:
I like that idea. I will come up with something more concise and see if @ochronus has a minute tomorrow to look it over with me.

Reply:
I'll have plenty of time in the afternoon, plz find me, happy to brainstorm :)